### DISCUSSION TITLE
what is the meaning of housenumbers in yellow?

### DISCUSSION THREAD

**Discussion by stephan75** [Zverik/every_door#402] [Q&A] [answered]
In the housenumber view, some housenumbers are displayed with white background, some others are with yellow background.

What is the meaning / difference?

Is this somewhere mentioned already and I missed it?

**Comment by Zverik** [comment, marked as answer]
Yellow background means a building has both `building:levels` and `roof:shape` tags set. This might change, see #396. Will add to the FAQ, #403.
